q2_feature_table/plugin_setup.py

```diff
@@ -469,7 +469,8 @@
         'method': 'Clustering methods exposed by seaborn (see http://seaborn.'
                   'pydata.org/generated/seaborn.clustermap.html#seaborn.clust'
                   'ermap for more detail).',
-        'cluster': 'Specify which axes to cluster.',
+        'cluster': 'Specify which axes to cluster. Option to cluster on '
```

Comment:
Can you drop the `Option to cluster on...` text? Interfaces will render the options out, instead:

```bash
# q2cli
  --p-cluster [features|both|samples]
                                  Specify which axes to cluster.  [default:
                                  both]
```

```python
# Artifact API
cluster : Str % Choices('features', 'both', 'samples'), optional
    Specify which axes to cluster.
```

Reply:
Sure thing.
To clarify, the q2cli and Artifact API options shown will be updated automatically based on the options available in `q2_feature_table/_heatmap/_visualizer.py`?

Reply:
Yep!

Reply:
QIIME 2 knows about those `Choices` via the method's registration: https://github.com/qiime2/q2-feature-table/blob/master/q2_feature_table/plugin_setup.py#L445

Reply:
Awesome, thanks for the link to the relevant code!